### PR TITLE
Improve Application Operator Tests

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Increase CPU of Istio Pilot on the local installation
- Improve tests of Application Operator

**Related issue(s)**
#3430 #3434

### CODE REVIEW

File: tests/application-gateway-tests/test/executor/proxy/suite.go [hunk 1]
```diff
@@ -215,6 +215,10 @@ func (ts *TestSuite) AddDenierLabel(t *testing.T, apiId string) {
 
 	pod.Labels[serviceName] = "true"
 
-	_, err = ts.podClient.Update(pod)
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```
Comment (aszecowka) [inline tests/application-gateway-tests/test/executor/proxy/suite.go]: I am not sure if this is correct usage of RetryOnConflict, please have a look here:
https://github.com/pingcap/tidb-operator/issues/81
Generally, we should fetch pod one more time, and again apply our changes.

Comment (Szymongib) [reply tests/application-gateway-tests/test/executor/proxy/suite.go]: Good point! I do not think it impacts current implementation but we should definitely correct it in the future.

**Pull request merged**